enhancements/arbiter-clusters.md

```diff
@@ -0,0 +1,628 @@
+---
+title: arbiter-clusters
+authors:
+  - "@eggfoobar"
+reviewers:
+  - "@tjungblu"
+  - "@patrickdillon"
+  - "@racedo"
+  - "@deads2k"
+  - "@jerpeter1"
+  - "@sjenning"
+  - "@yuqi-zhang"
+  - "@zaneb"
+  - "@rphillips"
+  - "@joelanford"
+approvers:
+  - "@jerpeter1"
+api-approvers:
+  - "@JoelSpeed"
+creation-date: 2024-08-27
+last-updated: 2024-10-24
+tracking-link:
+  - https://issues.redhat.com/browse/OCPEDGE-1191
+see-also: []
+replaces: []
+superseded-by: []
+---
+
+# Support 2 Node + 1 Arbiter Node HA Cluster
+
+## Summary
+
+This enhancement describes an ability to install OpenShift with a control plane
+that consists of at minimum 2 normal sized nodes, and at least 1 node that can
+be less powerful than the recommended node size. This 1 arbiter node will only
+be running critical components for maintaining HA to allow the arbiter node size
+to be as small and as low cost as possible with in reason.
+
+## Motivation
+
+Customers at the edge are requiring a more economical solution for HA
+deployments at the edge. They can support running 2 node clusters for redundancy
+but would like the option to deploy a lower cost node as an arbiter to supply
+the 3 nodes for ETCD quorum.
+
+### User Stories
+
+- As a solutions architect for a retail organization, I want to deploy OpenShift
+  at n number of store locations at the edge with only 2 regular sized nodes and
+  1 lower cost node to maintain HA and keep compute costs down.
+- As a solutions architect for cloud infrastructures, I want to offer low cost
+  OpenShift deployments on purpose built hardware for a 2 + 1 configuration.
+- As an OpenShift cluster admin I want non-critical applications deployed to my
+  2 + 1 arbiter node cluster to not be scheduled to run on the arbiter node.
+- As an OpenShift cluster admin I want to be able to allow deployments with
+  proper tolerations or explicitly defined node in the `spec` to be able to be
+  scheduled on the arbiter node.
+
+### Goals
+
+- Provide a new arbiter node role type that achieves HA but does not act as a
+  full master node.
+- HA for a 2+1 arbiter node should match the HA guarantees of a 3 Node Cluster
+  deployment.
+- Support installing OpenShift with 2 master nodes and 1 arbiter node.
+- Add a new ControlPlaneTopology and InfrastructureTopology in the
+  Infrastructure API
+- The arbiter node hardware requirements will be lower than regular nodes in
+  both cost and performance. Customers can use devices on the market from OEMs
+  like Dell that supply an all in one unit with 2 compute and 1 lower powered
+  compute for this deployment scenario.
+
+### Non-Goals
+
+The below goals are not intended to be worked on now, but might be expansion
+ideas for future features.
+
+- Running the arbiter node offsite.
+- Running a virtualized arbiter node on the same cluster.
+- Having a single arbiter supporting multiple clusters.
+- Moving from 2 + 1 to a conventional 3 node cluster
+
```

Comment:
- Supporting workers of type `arbiter`?

Reply:
No, thankfully, that's not in plan for this, the idea is that the Arbiter will function as a smaller less powerful Master, so there should never be an Arbiter that is a worker

Reply:
Added some more explicit wording around this so that's captured